get.sh

```diff
@@ -154,8 +154,8 @@ parseCommandLineArgs()
 		esac
 	done
 
-	# Check if TESTDIR exists and points to openjdk-tests
-	if [[ ! -d "$TESTDIR" || "$TESTDIR" != *"openjdk-tests"* ]]; then
+	# Check if TESTDIR exists and points to aqa-tests
+	if [[ ! -d "$TESTDIR" || "$TESTDIR" != *"aqa-tests"* ]]; then
 		echo "TESTDIR: $TESTDIR is invalid. Please use --testdir|-t to set valid TESTDIR under openjdk-tests. Default value current dir (pwd) is used if not provided."
```

Comment:
thanks @mbarbero - Please update the echo statement reference to openjdk-tests also